pkcs11test.cc

```diff
@@ -136,10 +137,12 @@ int main(int argc, char* argv[]) {
 
   // Retrieve PKCS module location.
   bool explicit_slotid = false;
+  bool use_slot_offset = false;
+  unsigned g_slot_offset = 0;
```

Comment:
This can just be `slot_offset` &ndash; the `g_` prefix is to mark a global.

Reply:
Yes sure thing